components/service/glean/src/main/java/mozilla/components/service/glean/ping/PingMaker.kt

```diff
@@ -111,9 +111,18 @@ internal class PingMaker(
      *        This usually matches with the name of the ping
      * @return a string holding the data for the ping.
      */
-    fun collect(storage: String): String {
+    fun collect(storage: String): String? {
         val jsonPing = storageManager.collect(storage)
+
+        // TODO, maybe need to not add the 'ping_info' to the jsonPing if the jsonPing is already empty
```

Comment:
The TODO says that we should not add `ping_info` to the JSON payload if the latter is empty. Am I interpreting this correctly? Because it seems to be what we're doing, since we're bailing out early when the length is null.

Reply:
Just me neglecting to clean up a comment.  I placed this here when I started on this bug to help identify all of the places I would need to make changes and I just didn't update this one to reflect the implementation.